app/views/articles/show.html.haml

```diff
@@ -5,4 +5,28 @@
         %p= @article.description
         %p= @article.content
         - if current_user == @article.user
-            = link_to 'Edit', edit_content_management_article_path(@article)
\ No newline at end of file
+            =link_to 'Edit', edit_content_management_article_path(@article)
+        
+        %h2 Comments
+        - @article.comments.each do |comment|
+            %p
+                %strong Commenter:
+                = comment.user.name
+            %p
+                %strong Comment:
+                = comment.body
+
+        - if user_signed_in?
```

Comment:
The feature name and user story says that only subscribers should be able to comment

Reply:
Indeed it does! Corrected the code now. Thanks.